boards/saml10-xpro/Makefile.features

```diff
@@ -0,0 +1,9 @@
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
```

Comment:
This is not needed and can be provided at CPU levels (several PRs were recently merged in that direction)

Reply:
I'll remove it. This is a leftover because this PR was started some times before this change was merged.